pkg/propertyprovider/aks/provider.go

```diff
@@ -29,10 +29,9 @@ import (
 )
 
 const (
-	// A list of properties that the AKS property provider collects.
+	// A list of properties that the AKS property provider collects in addition to the
```

Comment:
btw, I think we should always use Azure instead of AKS  as a provider

Reply:
https://secrets-store-csi-driver.sigs.k8s.io/concepts.html

Reply:
Fixed.